sdk/servicebus/service-bus/src/queueClient.ts

```diff
@@ -9,7 +9,12 @@ import { Client } from "./client";
 import { SessionReceiverOptions } from "./session/messageSession";
 import { Sender } from "./sender";
 import { Receiver, SessionReceiver } from "./receiver";
-import { throwErrorIfConnectionClosed } from "./util/utils";
+import {
+  getOpenReceiverErrorMsg,
```

Comment:
Would it be possible to have enum file for error codes/names too?
@bterlson thoughts ?

Reply:
Rather just have a module with exported constants/utilities than an enum, personally.

Reply:
The errors here are instances of the `Error` class.
The name is always "Error" and there is no code.
Please see the link @bterlson provided in https://github.com/Azure/azure-sdk-for-js/pull/2207#issuecomment-482682715 and my question that lead to that

Reply:
Got it, just curious - would guidelines apply to amqp-common-js package too?

In this case, I think moving to a new file `errors.ts` as opposed to `utils.ts` would be better to start with. We could move it out to a module later if necessary.

Reply:
I was considering having constants for the error messages, but since the messages can be dynamic due to inclusion of the type of client and the name of the entity, I ended up with helper utilities like these

Reply:
I like the idea of a separate file. Done with https://github.com/Azure/azure-sdk-for-js/pull/2207/commits/18a77ea8e1fa34d50f02f321716f3c884068a749

Reply:
> Got it, just curious - would guidelines apply to amqp-common-js package too?

Eventually, yes.